### DISCUSSION TITLE
Is there any up to date documentation?

### DISCUSSION THREAD

**Discussion by SomeoneElseOSM** [osmandapp/OsmAnd#13551] [Q&A]
### Description
I was trying to answer https://forum.openstreetmap.org/viewtopic.php?pid=852175#p852175 , since it's something that I've done myself lots of times before, and it's never been well documented.

It turns out that the process has changed (no idea why), and the new process involves choosing between two options ("add" and "install more") that basically mean exactly the same thing.  This prompted the more general question:

_**Is there any up-to-date, context-sensitive help for OsmAnd, and if so, where is it?**_

I'm aware of the "Help" link at the bottom of the hamburger menu in-app, but none of the options there are related to this question.  I know about https://osmand.net/help-online , and none of that applies either.  [searching](https://www.google.com/search?hl=en&q=layer%20site%3Ahttps%3A%2F%2Fosmand.net%2Fhelp%2Donline) finds a couple of pages such as https://osmand.net/help-online/technical-articles , but that doesn't apply either.

Specifically I'm not asking about a non-searchable list of "what are all the things you can do with OsmAnd" (the answer is "a lot").  What I'm asking about is "I want to do X, how do I do that?".

### How to reproduce?
Run OsmAnd
Be confused

### Your Environment
OsmAnd Version: 4.1.11
Android/iOS version: 11
Device model: Nokia X10

**Maps used (online or offline):**   
n/a

**Comment by vshcherb** [comment]
Here the latest documentation, hopefully soon will be all articles will be redirected https://docs.osmand.net/en

**Reply by SomeoneElseOSM** [reply]
Thanks - I'll check back later when that is complete.
